packages/p2p/src/stores/sendbird-store.js

```diff
@@ -270,7 +270,7 @@ export default class SendbirdStore extends BaseStore {
             } else {
                 this.markMessagesAsRead(true);
             }
-        }, 500);
+        }, 1000);
```

Comment:
As per SendBird documentation, `markAsRead` method has rate limit of one request per second. If it exceeds, SendBirdException will be thrown.